internal/provider/resources/flow.go

```diff
@@ -171,21 +167,17 @@ func (r *FlowResource) Create(ctx context.Context, req resource.CreateRequest, r
 
 	client, err := r.client.Flows(plan.AccountID.ValueUUID(), plan.WorkspaceID.ValueUUID())
 	if err != nil {
-		resp.Diagnostics.AddError(
-			"Error creating flows client",
-			fmt.Sprintf("Could not create flows client, unexpected error: %s. This is a bug in the provider, please report this to the maintainers.", err.Error()),
-		)
+		resp.Diagnostics.Append(helpers.CreateClientErrorDiagnostic("Flow", err))
+
+		return
```

Comment:
These are just cleanup changes to make use of the helpers we now have for adding diagnostics.